README.md

```diff
@@ -120,28 +90,7 @@ Now compile and upload to your device. If using a Heltec LoRa ESP32 board you sh
 
 If you don't see the captive portal screen, you can force it by accessing [neverssl.com](http://neverssl.com) which will force the captive portal to intercept the HTTP request.
```

Comment:
"which will help the captive portal intercept the HTTP request"

Reply:
solved